resources/js/header.js

```diff
@@ -0,0 +1,151 @@
+"use strict";
+
+var menuActive = false;
+var header = $('.header');
+setHeader();
+initCustomDropdown();
+initPageMenu();
+
+function setHeader()
+{
+
+    if(window.innerWidth > 991 && menuActive)
+    {
+        closeMenu();
+    }
+}
+
+function initCustomDropdown()
+{
+    if($('.custom_dropdown_placeholder').length && $('.custom_list').length)
+    {
+        var placeholder = $('.custom_dropdown_placeholder');
+        var list = $('.custom_list');
+    }
+
+    placeholder.on('click', function (ev)
+    {
+        if(list.hasClass('active'))
+        {
+            list.removeClass('active');
+        }
+        else
+        {
```

Comment:
```suggestion
        } else {
```
else cũng fix thế này nữa nha :) đọc lại hướng dẫn file convention